images/hub/dependencies

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+"""automatically manage requirements.txt dependencies with pip-tools
+
+./dependencies --help for info
```

Comment:
I'd appreciate some additional description here to summarize this.

Here are stuff I ended up writing down to try illustrate what I imagine could be useful to write here.

This script can identify and report how frozen representation of `requirements.in` that resides in `requirements.txt` is outdated. Also, when an update has been made to `requirements.in`, this script can update the frozen representation.   

This script is assumed to run in a folder with a `Dockerfile` file, a `requirements.in` file, a `requirements.txt`, and a `chartpress.yaml` file two folders up (`../../chartpress.yaml`) in order to function properly.